db/data.js

```diff
@@ -1103,9 +1096,9 @@ const produce = [
   {
     foodName: "Potato",
     imageURL:
-      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637899/Seasonal/057-organic-food_pikcaf.png",
+      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1666175321/Seasonal/026-potato_tent9c.png",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed feugiat euismod augue, ac fringilla diam pharetra id. In porttitor convallis nisi vel finibus. Pellentesque at diam venenatis, ornare mi eget, vestibulum arcu. Vivamus consectetur, erat eu semper dapibus, dolor sem suscipit mi, ut dignissim enim nibh vel risus. Sed aliquet, sapien eu congue gravida, lorem metus bibendum mauris, quis maximus nibh erat eu erat. Cras vulputate purus magna, tincidunt porttitor massa vestibulum sed. Nam turpis massa, dictum eu accumsan in, posuere sit amet lorem. Phasellus est sapien, faucibus eget elit et, blandit consequat ex. Proin feugiat arcu a magna iaculis, nec pharetra orci porta. Proin vel orci ante. Aliquam viverra non ligula nec rhoncus. Ut et tincidunt elit. Phasellus non risus in mi vulputate venenatis vitae quis enim. Vestibulum quis dolor venenatis, lobortis diam in, faucibus turpis. Nullam quis pharetra nulla, vestibulum tincidunt ipsum. Donec non ultrices ligula. Quisque at interdum velit. Donec laoreet elit enim, a vulputate risus cursus eu. Duis et est in mi pulvinar rhoncus in ut lorem. Integer nec dolor et velit sollicitudin vehicula. Duis sapien justo, condimentum nec orci eu, dictum dapibus nisl. Praesent ultricies libero vel leo pulvinar molestie.",
+      "Potatoes are an extremely popular root vegetable, with many varieties and many ways they can be cooked. They are technically a member of the nightshade family, like tomatoes and aubergines, and their shapes, colours and sizes can vary massively. Older potatoes should be scrubbed thoroughly before use, and any eyes should be removed, though otherwise, potatoes can be cooked with their skins on - unless you are boiling them for mash, in which case, it is better to peel them too. Any green spots should be removed from potatoes, as these bits will be mildly toxic, much like all above-ground plant growth on the original potato plants! Whether turned into fries, roasted, boiled, mashed, or added to stews, hashes, pies, or the hundreds of other recipes containing potato, they are a very versatile vegetable that you can do a lot with. Potatoes are available all year round, though different varieties are in season at different times. Potatoes can be grown at home with a fair amount of ease, though you will need a lot of soil and a very large, deep container in order to grow them. ",
```

Comment:
Extra space at the end of the string you renegade!

Reply:
Thank you!